doc/DataPlaneCodeGeneration/DeveloperDrivenEvolution.md

```diff
@@ -157,6 +187,7 @@ namespace Azure.AI.MetricsAdvisor
         // Can use same method name here
         public virtual async Task<Response<MetricFeedback>> CreateMetricFeedbackAsync(MetricFeedback feedback, CancellationToken cancellationToken = default)
         {
+            // We don't open a new scope here because it is just a wrapper
```

Comment:
Instead of "just a wrapper," let's say it's because 1) it has the same name as the protocol method, 2) it does not add any logic beyond serialization/deserialization and 3) no attributes are added to the span.